frontend/pages/hosts/ManageHostsPage/HostTableConfig.tsx

```diff
@@ -405,7 +405,7 @@ const allHostTableHeaders: IDataColumn[] = [
       return <HeaderCell value={titleWithToolTip} disableSortBy />;
     },
     disableSortBy: true,
-    accessor: "mdm_server_url",
+    accessor: "mdm_server_url", // TODO: should that be "mdm.server_url"?
```

Comment:
This should be confirmed by the frontend, but I believe the react-table way to do this would be to use an optional id  along with the accessor:
```
accessor: "mdm.server_url",
id: "mdm_server_url",
```